apps/wfo-ui/package.json

```diff
@@ -8,7 +8,8 @@
         "tsc": "tsc --noEmit",
         "build": "next build",
         "start": "next start",
-        "lint": "next lint"
+        "lint": "next lint",
+        "reset": "rm -rf node_modules"
```

Comment:
A reset script is not really needed. What would be the usecase?

Reply:
Do you know a simple way to remove all node_modules folders from all packages? So you can start clean (e.g. probably handy when we upgrade react or Next)